modulemd/v2/include/modulemd-2.0/modulemd-module-index-merger.h

```diff
@@ -161,6 +161,10 @@ modulemd_module_index_merger_associate_index (ModulemdModuleIndexMerger *self,
  * #ModulemdModuleIndexMerger is undefined. The only valid action on it after
  * that point is g_object_unref().
  *
+ * This function is equivalent to calling
+ * modulemd_module_index_merger_resolve_ext() with
+ * `strict_default_streams=False`.
```

Comment:
Here and throughout the newly added documentation, `False` &rarr; `FALSE` and `True` &rarr; `TRUE`.

Reply:
OK, I'll make that change and push this. Thanks!